oshc/main/static/main/js/script.js

```diff
@@ -5,4 +5,68 @@ $(document).ready(function() {
       scrollTop: 0
     }, 1000);
   });
+  //  code for PASSWORD STRENGTH METER
+  $(".strength-wrapper").css("visibility", "hidden"); //hide the strength meter initially
+  console.log("checker2");
+  $("#id_password1").bind("keyup", function() {
+    if ($(this).val().length == 0) {
```

Comment:
@bikrambhattacharya Fix server error
Actual behaviour:
![image](https://user-images.githubusercontent.com/8705386/28747564-c8edaffe-74bf-11e7-8c55-8110b7849393.png)

If you change debug to true then it's working (You can find this in `Settings.py`)
![image](https://user-images.githubusercontent.com/8705386/28747576-3245bad2-74c0-11e7-88f7-58d4d24bf831.png)
Also, Please don't change the code which is not relevant to PR (`Password Confirmation` should be on left)
Screenshot of the issue:
![image](https://user-images.githubusercontent.com/8705386/28747591-ab8ce398-74c0-11e7-8bbf-29ce59c0aeea.png)
the `weak` word should be bold and prefer different color which suits the theme and should be on the left
![image](https://user-images.githubusercontent.com/8705386/28747616-2943cacc-74c1-11e7-99dd-3faff6631e52.png)
There is space left to the right. Please make sure the strength indicator is matched to the parent.
and there should be `5px` space between strength and the `weak` word
@nikhita What are your suggestions?